docs/_sass/layout.scss

```diff
@@ -438,4 +436,32 @@ hr {
   .for-programs {
 	background: linear-gradient(to bottom, #395054 0%, #00D3A9 100%) no-repeat #a05fb7;
 
-  }
\ No newline at end of file
+  }
+
+  .nav-item{
+	list-style: none;
+	display: flex;
+	align-items: center;	
+	.nav-text {
+		font-size: 0.8rem !important;
```

Comment:
```suggestion
		font-size: 1.2rem !important;
```

Reply:
Upon increasing the font the icon gets shifted a bit towards the right. Make it stick to the center